Paltineanu Rares-Mihai/3.7 - curs3 V2.0/main.c

```diff
@@ -0,0 +1,20 @@
+///i hope that i did what you sugessted. Thanks!
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+int main()
+{
+
+    clock_t x, y;
+    float time;
+    int a;
+    x = clock();
+    while(scanf("%d",&a))
```

Comment:
Awesome job, finding the clock() function and the CLOCKS_PER_SEC constant

Good work